CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
+### 5.1.7.1.0
+- This version of the adapter supports Chartboost Mediation SDK version 5.+.
```

Comment:
`5.+` looks a bit weird especially when there's the period at the end of the sentence. I'd just write it all out: `version 5.0 and later`.

Reply:
oops I'm halfway through merging. you're doing the update to latest, right? Feel free to update.